### PR TITLE
fix(ui): updated the link of scheduler documentation

### LINKED ISSUE #1950: Broken link in Upload instructions of scheduler documentation
### Description

In Upload > Instructions the scheduler documentation is redirecting to the broken link.

#### How to reproduce

/src/www/ui/template/upload_instructions.html.twig line 82
`<a href="http://www.fossology.org/projects/fossology/wiki/Sched-20">{{ 'Scheduler documentation'| trans }}</a>.`

#### Screenshots

Screenshot from 2021-04-05 17-10-12

### PR DESCRIPTION
**Pull request by Shruti3004**
Closes #1950 

## How to test

Uploads>instructions>scheduler docmentation

### CODE REVIEW

**Comment by Shruti3004** [comment]
@ag4ums please review

File: src/www/ui/template/upload_instructions.html.twig [hunk 1]
```diff
@@ -79,5 +79,5 @@
   <em>
     {{ 'Configuring the Scheduler in the '| trans }}
   </em>
-  <a href="http://www.fossology.org/projects/fossology/wiki/Sched-20">{{ 'Scheduler documentation'| trans }}</a>.
+  <a href="https://github.com/fossology/fossology/wiki">{{ 'Scheduler documentation'| trans }}</a>.
```
Comment (GMishx) [inline src/www/ui/template/upload_instructions.html.twig:82]: Does using this link makes more sense?
https://github.com/fossology/fossology/wiki/Job-Scheduler

Comment (Shruti3004) [reply src/www/ui/template/upload_instructions.html.twig:82]: okay @GMishx i'll update it with this

**Comment by Shruti3004** [comment]
@GMishx now please check

**Comment by GMishx** [comment]
Oh, it was just a question per say. But looks good.

Can you please squash the commits for merging?

**Comment by Shruti3004** [comment]
> Oh, it was just a question per say. But looks good.
> 
> Can you please squash the commits for merging?

yes for sure

**Comment by Shruti3004** [comment]
@GMishx I have squashed it.

**Review by GMishx — APPROVED** [review]
Changes looks good.

**Pull request merged**